var/spack/repos/builtin/packages/lucene/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Lucene(Package):
+    """
+    Apache Lucene is a high-performance, full featured text search engine
+    library written in Java.
+    """
+
+    homepage = "https://lucene.apache.org/"
+    url      = "http://archive.apache.org/dist/lucene/java/8.3.1/lucene-8.3.1.tgz"
+    list_url = "http://archive.apache.org/dist/lucene/java/"
+    list_depth = 2
```

Comment:
I think this should be 1?

Reply:
OK, looks like My Spack Document is out of date.

Reply:
Fixed it in https://github.com/spack/spack/pull/14316/commits/17558d11c2943e55b9ae390401cfa097798e360d